src/python/WMCore/Storage/Backends/GFAL2Impl.py

```diff
@@ -113,18 +144,37 @@ def buildCopyCommandDict(self, sourcePFN, targetPFN, options=None, checksums=Non
 
         return copyCommandDict
 
-    def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=None):
+    def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=None, auth_method=None, force_method=False):
         """
         Create gfal-cp command for stageOut
 
         :sourcePFN: str, PFN of the source file
         :targetPFN: str, destination PFN
         :options: str, additional options for gfal-cp
         :checksums: dict, collect checksums according to the algorithms saved as keys
+        :auth_method: str, the authentication method to be used ("X509", "TOKEN", or None)
```

Comment:
@anpicci similar to above, either you describe both auth_method and force_method, or you dont mention neither here. maybe it's enough to describe them only in the `adjustSetup()` function, and not everywhere else

Reply:
@mapellidario for clarity, I decided to describe them both here and in `adjustSetup`